PowerShell/JumpCloud Module/JumpCloud.psm1

```diff
@@ -12,6 +12,10 @@ Foreach ($Import in @($Public + $Private))
         Write-Error -Message "Failed to import function $($Import.FullName): $_"
     }
 }
+
+# Check to see if parallel processing is available for the session
+$global:JCParallel = Get-JCParallelValidation
```

Comment:
Just a note for later, we'll be calculating this value with [SA-2727](https://jumpcloud.atlassian.net/browse/SA-2727)